zork.js

```diff
@@ -0,0 +1,329 @@
+//globals
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        readlineInterface.question(questionText, resolve);
+    });
+}
+
+class Player{
+    constructor(name,title){
+        this.name = name,
+        this.title = title
+    }
+    inventory = [];
+    drop = function(item){
+        delete this.inventory[item];
```

Comment:
this is an excellent, and very concise way of removing an item from an inventory.  I kind of wish I'd done it this way when I was writing my code...